src/main/java/at/hannibal2/skyhanni/utils/renderables/Renderable.kt

```diff
@@ -358,10 +408,33 @@ interface Renderable {
             override val height = render.height
             override val horizontalAlign = horizontalAlign
             override val verticalAlign = verticalAlign
-
             override fun render(posX: Int, posY: Int) {
                 render.renderXAligned(0, 0, width)
             }
         }
+
+        fun horizontalContainer(
```

Comment:
this very function just without spacing does already exist in the class. please merge the functions.